File: SAMPLES/CPP/hello.cpp
/* hello.cpp - Variables, constantes, chaines, cout */
#include <iostream>

const int MAX = 10;

int main() {
    int x = 42;
    int y = 100;
    const char* msg = "Hello, World!";

    cout << msg << endl;
    cout << "x = " << x << endl;
    cout << "y = " << y << endl;

    int sum = x + y;
    cout << "sum = " << sum << endl;

    return 0;
}

File: SAMPLES/JS/hello.js
// hello.js - Exemple JS86 : console.log, var/let/const, +, template literals
// Compile : JS86.EXE hello.js hello.asm

// Variables et constantes
var message = "Hello";
let world = "World";
const bang = "!";

// Concatenation de chaines
var greeting = message + " " + world + bang;
console.log(greeting);

// Affichage de nombres
var x = 42;
let y = -7;
console.log(x);
console.log(y);

// Expressions arithmetiques
var sum = x + y;
console.log(sum);

// Template literal basique
var name = "JS86";
console.log(`Bienvenue dans ${name}`);

// Constante booleenne
const flag = true;
console.log(flag);

// Valeurs speciales
console.log(null);
console.log(undefined);
